db/migrate/20250218070141_add_registered_at_timestamp.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class AddRegisteredAtTimestamp < ActiveRecord::Migration[7.2]
+  def up
+    add_column :registrations, :registered_at, :datetime, precision: 6, default: -> { 'NOW(6)' }
```

Comment:
Also, not happy about a low-level default value :/ Is there really no other way?

Reply:
Still curious though about the default value. I'd rather mark this as non-null and backfill accordingly.
(I think if you create the whole column as non-null, it will get the Unix epoch or some other silly default, but then you backfill with actual contents directly below, so we're good)

Reply:
Sorry just didn't get around to changing this! That was just a placeholder to populate the rows with a timestamp while we decided how to populate it. Will be replaced by an explicitly-defined registered_at timestamp per our Slack discussin

Reply:
That should be better now :)